src/odemis/acq/test/stream_test.py

```diff
@@ -30,17 +30,18 @@
 import odemis
 from odemis.acq import stream, calibration, path, leech
 from odemis.acq.leech import ProbeCurrentAcquirer
-from odemis.acq.stream import Stream
+from odemis.acq.stream import Stream  # TODO not needed??
 from odemis.dataio import tiff
 from odemis.driver import simcam
 from odemis.util import test, conversion, img
 import os
 import threading
 import time
 import unittest
-from unittest.case import skip
+from unittest.case import skip  # TODO not needed??
```

Comment:
Usually, we have a lot of `#@skip`, which allow to skip a test just by uncommenting the line, so leave this import here.